backend/gzdb/db.go

```diff
@@ -715,6 +715,35 @@ func (g *gormZenaoDB) GetEventTickets(eventID string) ([]*zeni.SoldTicket, error
 	return res, nil
 }
 
+// GetEventCommunity implements zeni.DB.
+func (g *gormZenaoDB) GetEventCommunity(eventID string) (*zeni.Community, error) {
+	evtIDInt, err := strconv.ParseUint(eventID, 10, 64)
+	if err != nil {
+		return nil, fmt.Errorf("parse event id: %w", err)
+	}
+
+	var roles []EntityRole
+	if err := g.db.
+		Where("entity_type = ? AND entity_id = ? AND org_type = ? AND role = ?",
+			zeni.EntityTypeEvent, evtIDInt, zeni.EntityTypeCommunity, zeni.RoleEvent).
+		Find(&roles).Error; err != nil {
+		return nil, err
+	}
+	if len(roles) == 0 {
+		return nil, nil
+	}
+	if len(roles) > 1 {
+		return nil, fmt.Errorf("event %d has multiple communities: %d", evtIDInt, len(roles))
+	}
```

Comment:
it should not happen but in case it happens i added this to flag it easily since it's an unexpected behavior